### PR TITLE
Add "HLS" section to <amp-video>

### PR DESCRIPTION
**Pull request by ithinkihaveacat**
(My reminder: once deployed, use it to answer [this Stack Overflow question](https://stackoverflow.com/q/48101141/11543).)

### CODE REVIEW

File: src/20_Components/amp-video.html [hunk 1]
```diff
@@ -62,6 +62,31 @@
     <source type="video/webm" src="/video/tokyo.webm">
   </amp-video>
 
+  <!-- ## HLS -->
+  <!--
+    AMP supports adaptive bitrate video via
+    [HLS](https://en.wikipedia.org/wiki/HTTP_Live_Streaming) provided it is
+    supported by the browser. (Note that whilst HLS is supported by most mobile
+    browsers (including Chrome and Safari), few desktop browsers support it.)
```
Comment (sebastianbenz) [inline src/20_Components/amp-video.html]: Can you avoid nested parentheses?

File: src/20_Components/amp-video.html [hunk 2]
```diff
@@ -62,6 +62,31 @@
...
+    
+    (The videos below have the bitrate burnt into the video files themselves to
+    make it easier to discover when the browser switches between streams. If
+    your browser does not support HLS, you will see "No HLS". If you need to
+    create files in the appropriate format, we recommend investigating [shaka
+    packager](https://github.com/google/shaka-packager).)
```
Comment (sebastianbenz) [inline src/20_Components/amp-video.html]: Remove parentheses ;-)

**Comment by ithinkihaveacat** [comment]
Fixed!

**Comment by sebastianbenz** [comment]
Thanks!!!

**Pull request merged**